docs/calicoctl/pool.md

```diff
@@ -0,0 +1,145 @@
+
+# User guide for 'calicoctl pool' commands
+> NOTE: The `calicoctl pool` configuration commands should NOT be used when 
+> running  Calico with the Docker libnetwork driver.  The libnetwork driver 
+> manages the IP address pools and assignment.  
+
+This sections describes the `calicoctl pool` commands.
+
+These commands allow users to define and view IP address pools from which endpoint 
+IP addresses are allocated.  Users can add, remove, update, or view the IP pools.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a 
+full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl pool' commands
+
+Run `calicoctl pool --help` to display the following help menu for the 
+calicoctl pool commands.
+
+```
+
+Usage:
+  calicoctl pool add <CIDRS>... [--ipip] [--nat-outgoing]
+  calicoctl pool remove <CIDRS>...
+  calicoctl pool range add <START_IP> <END_IP> [--ipip] [--nat-outgoing]
+  calicoctl pool show [--ipv4 | --ipv6]
+
+Description:
+  Configure IP Pools
+
+Options:
+  --ipv4          Show IPv4 information only
+  --ipv6          Show IPv6 information only
+  --nat-outgoing  Apply NAT to outgoing traffic
+  --ipip          Use IP-over-IP encapsulation across hosts
+ 
+```
+
+## calicoctl pool commands
+
+
+### calicoctl pool add
+This command is used to add or modify subnet CIDR pools in Calico.
+
+The command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl pool add <CIDRS>... [--ipip] [--nat-outgoing]
+
+    <CIDRS>: A single or list of subnet cidrs separated by spaces.
+
+    --ipip: Use IP-over-IP encapsulation across hosts.
+    --nat-outgoing: Apply a NAT to outgoing traffic.
+```
+
+When a container is added to Calico networking (using Docker default networking), 
```

Comment:
It's broader than this.  Any time Calico IPAM is in use, including Docker default networking, Mesos, Kubernetes (when Calico IPAM is enabled), then we use the pools to know the range of addresses available.